File: 프로그래머스/0/181903. qr code/qr code.js
function solution(q, r, code) {
    const codeArr = code.split("");
    const result = [];
    for(let i = 0; i < codeArr.length; i++){
            if(i % q === r){
                result.push(codeArr[i])
            }
        }
return result.join("")
}

File: 프로그래머스/0/181903. qr code/qr code.py
# q = 3
# r = 1
# code = "qjnwezgrpirldywt"

def solution(q, r, code):
    result = ""
    for i in range(len(code)):
        if i % q == r:
            # print("나머지가 r인 위치",i,q,r)
            # print("-->",code[i])
            result += code[i]
    return result